src/Components/Categories/CategoriesFunctions.js

```diff
@@ -0,0 +1,30 @@
+import * as serviceTypes from './CategorieServices'
```

Comment:
this changes are not scope of this user story, please delete this file from this PR

Reply:
Done!